superdesk/publish/subscribers.py

```diff
@@ -107,8 +109,6 @@ class SubscribersResource(Resource):
         }
     }
 
-    datasource = {'default_sort': [('_created', -1)]}
```

Comment:
Sorry, I just noticed in the superdesk PR that you do the sorting from the client side.